RELEASES.md

```diff
@@ -22,11 +27,12 @@ version from the `main` branch by creating a `vX.Y.0` tag and a corresponding `r
 branch, with merge permissions given to the release manager of stable releases, and CI configured
 to execute tests on it.
 
-### Security releases
+### Security Releases
```

Comment:
nit: prefer normal capitalization for titles

there is no ultimate authority on this so its a matter of style. Generally i think normal capitalization is preferable for the same reasons as in normal prose - it allows capitalization for proper names etc

also capitalizing then throws up more style issues - eg do and/or/if/etc get capitalized

Reply:
fwiw I agree with @phlax